networkx/algorithms/operators/tests/test_unary.py

```diff
@@ -40,6 +40,23 @@ def test_complement_2():
                   ('B', 'D'), ('C', 'A'), ('C', 'B'),
                   ('C', 'D'), ('D', 'A'), ('D', 'B'), ('D', 'C')])
 
+def test_mycielski():
+    G=nx.path_graph(2)
+    M=mycielski(G)
+    assert_true(is_isomorphic(M, cycle_graph(5)))
+
+def test_mycielski_2():
```

Comment:
The name of these test methods should reflect their purpose. For example, this one might be called `test_mycielski_size()`.